apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -52,6 +52,19 @@
                 <%= link event_teaser.title, to: path_fn.() %>
               </div>
             </div>
+            <div>
```

Comment:
ok!  The only one they really have in common is the margin, because the button doesn't have a text-align or width set, but is determined by its other properties.  But they definitely do have margin in common